.github/workflows/node.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
       - name: Code Review
-        uses: nishtahir/conventional-gpt@main
+        uses: nishtahir/conventional-gpt@develop
```

Comment:
Changing the tagged version from `main` to `develop` should be done cautiously. Ensure that the `develop` branch is stable and thoroughly tested, as this could introduce instability into the codebase. If not already done, please verify the stability and compatibility of the `develop` branch before merging.